jenkins_job_wrecker/modules/buildwrappers.py

```diff
@@ -143,26 +143,27 @@ def prebuildcleanup(top, parent):
     preclean = {}
     preclean_patterns = {'include': '', 'exclude': ''}
     for element in top:
-        if element.tag == 'deleteDirs':
-            preclean['dirmatch'] = (element.text == 'true')
-        elif element.tag == 'patterns':
-            for subelement in element:
-                if subelement.tag != 'hudson.plugins.ws__cleanup.Pattern':
-                    raise NotImplementedError("cannot handle "
-                                              "XML %s" % subelement.tag)
-                if subelement.find('type') is not None and subelement.find('pattern') is not None:
-                    rule_type = subelement.find('type').text.lower()
-                    rule_patt = subelement.find('pattern').text
-                    preclean_patterns[rule_type] = rule_patt
-        elif element.tag == 'cleanupParameter':
-            preclean['check-parameter'] = element.text
-        elif element.tag == 'externalDelete':
-            preclean['external-deletion-command'] = element.text
-        elif element.tag == 'disableDeferredWipeout':
-            preclean['disable-deferred-wipeout'] = (element.text == 'true')
-        else:
-            raise NotImplementedError("cannot handle "
-                                      "XML %s" % element.tag)
+        if element.text is not None:
```

Comment:
You could:

    if element.text is None:
        continue

and leave the rest untouched?

Reply:
Done. Thanks.